pkg/util/languagedetection/annotations.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
Same here, this code is not shared and purely functional, it should go to `pkg/cluster/languagedetection`